api/researches/views.py

```diff
@@ -87,7 +88,10 @@ def get_researches(request):
 @group_required("Оператор", "Конструктор: Параклинические (описательные) исследования", "Врач стационара")
 def researches_by_department(request):
     direction_form = DResearches.DIRECTION_FORMS
-    response = {"researches": [], "direction_forms": direction_form}
+    spec = users.Speciality.objects.all().order_by("title")
+    spec_data = [{"pk": s.pk, "title": s.title} for s in spec]
+    spec_data.append({"pk": -1, "title": "Не выбрано"})
```

Comment:
Обычно мы делаем "Не выбрано" в самом верху и одной строкой
```suggestion
    spec_data = [{"pk": -1, "title": "Не выбрано"}, *list(users.Speciality.objects.all().values('pk', 'title').order_by("title"))]
```